bcda/models/models.go

```diff
@@ -220,27 +220,34 @@ func (aco *ACO) GetBeneficiaries(includeSuppressed bool) ([]CCLFBeneficiary, err
 		return cclfBeneficiaries, fmt.Errorf("unable to find cclfFile")
 	}
 
-	if err := db.Find(&cclfBeneficiaries, "file_id = ?", cclfFile.ID).Error; err != nil {
+	var suppressedHICNs []string
+	if !includeSuppressed {
+		db.Raw(`SELECT DISTINCT s.hicn
+			FROM (
```

Comment:
Nice!! I was able to step through this visually and I'm pretty sure it works. I think we need to add another check (either automated or manual) to make sure suppressed benes aren't included in the final payload.

Reply:
This test was written with the original suppression logic and still passes with the changes: https://github.com/CMSgov/bcda-app/blob/master/bcda/models/models_test.go#L659